src/visualizr/anitalker/experiment.py

```
@@ -340,0 +1,343 @@
+        max_steps=conf.total_samples // conf.batch_size_effective,
+        resume_from_checkpoint=resume,
+        gpus=gpus,
+        num_nodes=nodes,
+        accelerator=accelerator,
+        precision=16 if conf.fp16 else 32,
+        callbacks=[
+            checkpoint,
+            LearningRateMonitor(),
+        ],
+        # clip in the model instead
+        # gradient_clip_val=conf.grad_clip,
```

Ruff (0.12.2)

Found commented-out code

Remove commented-out code

(ERA001)